### PR TITLE
Translated install-kubesphere-on-qingcloud-vms-zh.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 1]
```diff
@@ -1,153 +1,153 @@
 ---
-title: "Deploy KubeSphere on QingCloud Instances"
-keywords: "KubeSphere, Installation, HA, High-availability, LoadBalancer"
-description: "The tutorial is for installing a high-availability cluster on QingCloud instances."
-linkTitle: "Deploy KubeSphere on QingCloud Instances"
+title: "在青云QingCloud主机上部署 KubeSphere"
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:2]: 在青云QingCloud主机上
->
在青云QingCloud 主机上

"青云QingCloud" is the company name so no space is needed between “青云” and "QingCloud", while you need to add a space between "QingCloud" and Chinese characters that come after it. Please change it across the guide.

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 2]
```diff
@@ -1,153 +1,153 @@
...
+### 步骤 1：创建负载均衡器
 
-This step demonstrates how to create load balancers on the QingCloud platform.
+本步骤演示如何在青云QingCloud平台上创建负载均衡器。
 
-#### Create an internal load balancer
+#### 创建内部负载均衡器
 
-1. Log in the [QingCloud console](https://console.qingcloud.com/login). In the menu on the left, under **Network & CDN**, select **Load Balancers**. Click **Create** to create a load balancer.
+1. 登录[青云QingCloud控制台](https://console.qingcloud.com/login)。在左侧导航栏选择**网络与 CDN **下的**负载均衡器**，然后点击**创建**。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:41]: **网络与 CDN **下的**负载均衡器**
->
**网络与 CDN** 下的**负载均衡器**

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 3]
```diff
@@ -1,153 +1,153 @@
...
+   ![create-lb](/images/docs/zh-cn/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/create-lb.png)
 
-2. In the pop-up window, set a name for the load balancer. Choose the VxNet where your machines are created from the **Network** drop-down list. Here is `pn`. Other fields can be default values as shown below. Click **Submit** to finish.
+2. 在弹出的对话框中，设置负载均衡器的名称，在**网络**下拉列表中选择机器所在的私有网络（在本例中为 `pn`），其他参数可以保持默认，然后点击**提交**。 
 
-   ![qingcloud-lb](/images/docs/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/qingcloud-lb.png)
+   ![qingcloud-lb](/images/docs/zh-cn/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/qingcloud-lb.png)
 
-3. Click the load balancer. On the detail page, create a listener that listens on port `6443` with the **Listener Protocol** set to `TCP`.
+4. 点击上一步创建的负载均衡器。在其详情页面创建监听器，将**监听协议**设置为**TCP**，将**端口**设置为 `6443`。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:49]: 将**监听协议**设置为**TCP**
->
将**监听协议**设置为 `TCP`

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 4]
```diff
@@ -1,153 +1,153 @@
...
+3. 点击**添加后端**。在弹出的对话框中选择私有网络 `pn`，点击**高级搜索**，选择私有网络 `pn` 中的六台机器用于安装 KubeSphere，并将**端口**设置为 `30880`。
 
-   ![six-instances](/images/docs/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/six-instances.png)
+   ![six-instances](/images/docs/zh-cn/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/six-instances.png)
 
-   Click **Submit** when you finish.
+   设置完成后点击**提交**。
 
-4. Click **Apply Changes** to use the configurations. At this point, you can find the six machines have been added as the backend servers of the listener that is behind the external load balancer.
+4. 点击**应用修改**。页面上显示六台机器已添加为外部部负载均衡器后端监听器的后端服务器。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:114]: 外部部负载均衡器
->
外部负载均衡器

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 5]
```diff
@@ -1,153 +1,153 @@
...
-Follow the step below to download KubeKey.
+请按照以下步骤下载 KubeKey。
 
 {{< tabs >}}
 
-{{< tab "Good network connections to GitHub/Googleapis" >}}
+{{< tab "如果您能正常访问 GitHub/Googleapis" >}}
 
-Download KubeKey from its [GitHub Release Page](https://github.com/kubesphere/kubekey/releases) or use the following command directly.
+从 [GitHub 发布页面](https://github.com/kubesphere/kubekey/releases) 下载 KubeKey 或直接使用以下命令：
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:126]: 从 [GitHub 发布页面](https://github.com/kubesphere/kubekey/releases) 下载 KubeKey 或直接使用以下命令：
->
从 [GitHub 发布页面](https://github.com/kubesphere/kubekey/releases)下载 KubeKey 或直接使用以下命令：

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 6]
```diff
@@ -243,53 +241,53 @@ respectively, and you can refer to the following example.
...
 
-- The address and port should be indented by two spaces in `config-sample.yaml`, and the address should be VIP.
-- The domain name of the load balancer is `lb.kubesphere.local` by default for internal access. If you need to change the domain name, uncomment and modify it.
+- 在 `config-sample.yaml` 文件中，`address` 和 `port` 字段应缩进两个空格，同时 `address` 字段的值应为 VIP 地址。
+- 负载均衡器的默认域名为 `lb.kubesphere.local`，用于内部访问。您可以在 `domain` 字段修改域名。
 
 {{</ notice >}}
 
-### Step 5: Kubernetes cluster configurations (Optional)
+### 步骤5：配置 Kubernetes 集群（可选）
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:249]: 步骤5
->
步骤 5

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 7]
```diff
@@ -243,53 +241,53 @@ respectively, and you can refer to the following example.
...
 
-Kubekey provides some fields and parameters to allow the cluster administrator to customize Kubernetes installation, including Kubernetes version, network plugins and image registry. There are some default values provided in `config-example.yaml`. You can modify Kubernetes-related configurations in the file based on your needs. For more information, see [Kubernetes Cluster Configurations](../../../installing-on-linux/introduction/vars/).
+集群管理员可修改 KubeKey 提供的一些字段来自定义 Kubernetes 安装参数，包括 Kubernetes 版本、网络插件和镜像仓库。 `config-example.yaml` 文件中的一些字段有默认值。您可以根据需要修改文件中 Kubernetes 相关的字段。有关更多信息，请参考[ Kubernetes 集群配置](../../../installing-on-linux/introduction/vars/)。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:251]: 网络插件和镜像仓库。 `config-example.yaml` 文件中
->
网络插件和镜像仓库。`config-example.yaml` 文件中

Delete the extra space.

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 8]
```diff
@@ -317,28 +315,28 @@ https://kubesphere.io             2020-08-13 10:50:24
...
+### 步骤 10：验证高可用集群
 
-Now that you have finished the installation, go back to the detail page of both the internal and external load balancers to see the status.
+安装完成后，打开内部和外部负载均衡器的详情页面查看节点状态。
 
-![active](/images/docs/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/active.png)
+![active](/images/docs/zh-cn/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/active.png)
 
-Both listeners show that the status is **Active**, meaning nodes are up and running.
+如果两个监听器中的节点状态都是**活跃**，所有节点已启动并运行正常。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:324]: 所有节点已启动并运行正常。
->
表明所有节点已启动并运行正常。

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md [hunk 9]
```diff
@@ -1,153 +1,153 @@
...
+   ![create-lb](/images/docs/zh-cn/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/create-lb.png)
 
-2. In the pop-up window, set a name for the load balancer. Choose the VxNet where your machines are created from the **Network** drop-down list. Here is `pn`. Other fields can be default values as shown below. Click **Submit** to finish.
+2. 在弹出的对话框中，设置负载均衡器的名称，在**网络**下拉列表中选择机器所在的私有网络（在本例中为 `pn`），其他参数可以保持默认，然后点击**提交**。
 
-   ![qingcloud-lb](/images/docs/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/qingcloud-lb.png)
+   ![qingcloud-lb](/images/docs/zh-cn/installing-on-linux/installing-on-public-cloud/deploy-kubesphere-on-qingcloud-instances/qingcloud-lb.png)
 
-3. Click the load balancer. On the detail page, create a listener that listens on port `6443` with the **Listener Protocol** set to `TCP`.
+4. 点击上一步创建的负载均衡器。在其详情页面创建监听器，将**监听协议**设置为`TCP`，将**端口**设置为 `6443`。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-qingcloud-vms.md:49]: 设置为`TCP`
->
设置为 `TCP`

Add a space

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Pull request merged**